pkg/debug/spec.sh

```diff
@@ -81,7 +83,10 @@ cat <<__EOT__
   "attr": {
     "memory": "${MEM}G",
     "storage": "${DISK}G",
-    "Cpus": "${CPUS}"
+    "Cpus": "${CPUS}",
+    "watchdog": ${WDT},
+    "hsm": ${HSM},
+    "leds": 0
```

Comment:
Was the intent to look in /sys/class/leds/ just to return the count of leds?

Reply:
The problem with looking there is that most of the time you'll get confused by the LEDs that are present on the keyboard (sine they get registered there) and by things like mmcX leds (which are not really visible). Hence for now this is just a placeholder for a human being to edit it.